specification/translation/Azure.AI.DocumentTranslation/client.tsp

```diff
@@ -41,49 +41,164 @@ interface SingleDocumentTranslationClient {
   documentTranslate is DocumentTranslation.DocumentTranslationOperations.documentTranslate;
 }
 
-// model overrides
-@@access(DocumentTranslation.DocumentTranslateResult,
-  Access.internal,
+
+@@clientName(DocumentTranslation.DocumentTranslateBody.body,
+  "documentTranslateContent",
+  "csharp,java"
+);
```

Comment:
I dont believe this is valid for this decorator. You can to separate them out, like:
```suggestion
@@clientName(DocumentTranslation.DocumentTranslateBody.body,
  "documentTranslateContent",
  "java"
);
@@clientName(DocumentTranslation.DocumentTranslateBody.body,
  "documentTranslateContent",
  "csharp"
);
```

Reply:
Same anywhere else this was done

Reply:
The diff is misleading. I didn't make these changes affecting csharp/java. Not sure if it is working for them, but I can update to separate them if you think it needs to be done.

Reply:
Yeah if you have the time, then you can separate them. As far as I'm aware this format doesnt work with this decorator.